### DISCUSSION TITLE
31st August 2022 - MSF Delegate Sync

### DISCUSSION THREAD

**Discussion by futuristudios** [omigroup/MSF-Delegates#14] [General]
MSF Delegate Sync -
---
_Wednesday 31st August 2022. (1500hrs UTC)_

the following is a summary of items discussed at this MSF (Metaverse Standards Forum) delegates sync call.
this is happening after the [Metaverse Standards Forum: Oversight (EMEA) #6](https://github.com/omigroup/MSF-Delegates/discussions/6) call.

discussed seeking clarity on the purpose of the delegates.
being the connective tissue between OMI and MSF.
need to be aware of subjective opinion and objective relaying of pertinent information.
feedback directionality/frequency - delegate from MSF to OMI occasionally vice versa.
part of the how was expressed as the need to have accommodating asynchronicity.

we set the kanban as public.

steps to getting MSF info via calendar:
1. create OMI_username
2. join groups (which ones? who's joined which - we'll need to discuss)
3. subscribe to calendars.
4. for delegates: attend calls, take notes, discuss, relay to community e.t.c

 - draft the [agenda for next week](https://github.com/omigroup/MSF-Delegates/discussions/16) [^1]

---
[^1]: _next meeting_: Wednesday 7th September 2022. (1500hrs UTC)

**Comment by funksoup** [comment]
Next week: 
Dive back into [MSF brainstorm/scope of work miro board](https://miro.com/app/board/uXjVPe7tenM=/) to better define mission/scope of work and relate back to kanban items.

**Comment by pcsmgnt** [comment]
**_Agenda:_**

15:00 (UTC):  Miro Board Brainstorm (as per above)
15:30 (UTC): Jesse Alton Github Repo Tips and Tricks
